prober/derp.go

```diff
@@ -577,6 +580,196 @@ func runDerpProbeNodePair(ctx context.Context, from, to *tailcfg.DERPNode, fromc
 	return nil
 }
 
+// derpProbeBandwidthMu ensures that derpProbeBandwidthTUN doesn't run concurrently.
+// This is necessary to avoid conflicts trying to create the TUN device.
```

Comment:
We could in theory create multiple TUN devices, one per test run, with unique names and addresses. I like the simplicity of making it consecutive, though. I also appreciate how this synchronization means we only consume bandwidth for one test path at a time, which is desirable for now. Perhaps expand this comment to include that info.